tests/api/test_api.py

```diff
@@ -110,3 +114,42 @@ def todo_test_get_runs():
     res_json = response.json()
     assert response.status_code == 200
     assert res_json['data'][0][0] == str(pid)
+
+
+
+def test_hogDB_add():
+    hog_data_obj  = [
+    {
+        "time": 1710668240000,
+        "data": hog_data.hog_string,
+        "settings": json.dumps({"powermetrics": 5000, "upload_delta": 3, "upload_data": True, "resolve_coalitions": ["com.googlecode.iterm2", "com.apple.terminal", "com.vix.cron"], "client_version": "0.5"}),
```

Comment:
@ribalba  can we use orjson here?

Reply:
Yes, what would be the improvement?

Reply:
I just wanted to fix this. Why do you want to use orjson?